package/MDAnalysis/transformations/nojump.py

```diff
@@ -99,11 +99,8 @@ def __init__(
         self,
         check_continuity=True,
         max_threads=None,
-        # NoJump transforms are inherently unparallelizable, since
-        # it depends on the previous frame's unwrapped coordinates
-        parallelizable=False,
     ):
-        super().__init__(max_threads=max_threads, parallelizable=parallelizable)
+        super().__init__(max_threads=max_threads, parallelizable=False)
```

Comment:
Keep the comment — this is valuable information!!
```suggestion
       # NoJump transforms are inherently unparallelizable, since
       # it depends on the previous frame's unwrapped coordinates
       super().__init__(max_threads=max_threads, parallelizable=False)
```